setup.py

```diff
@@ -42,5 +36,5 @@
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
-    python_requires=">=3.9,<3.13",
+    python_requires=">=3.10,<3.13",
```

Comment:
I'm assuming this was probably because of the previous constraint in uagents/uagents-core.
```suggestion
    python_requires=">=3.10,<4.0",
```